backend/src/main/java/com/wpz/rbs/controller/ActivityController.java

```diff
@@ -0,0 +1,35 @@
+package com.wpz.rbs.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.wpz.rbs.model.Activity;
+import com.wpz.rbs.service.ActivityService;
+
+@RestController
+@RequestMapping("activity") 
+public class ActivityController {
+    
+    @Autowired
+    ActivityService activityService;
+
+    @GetMapping()
+    private List<Activity> getAll(){
+        return activityService.getAll();
+    }
+
+    @GetMapping("{id}")
```

Comment:
Error when bad id and problem with quering by full id (looks like it is dividing in link)